### DISCUSSION TITLE
Fooocus + Nvidia 4080 instead Colab+?

### DISCUSSION THREAD

**Discussion by quasar136a8** [lllyasviel/Fooocus#2330] [Q&A] [answered]
HI, I have a question about the graphic card Nvidia GeForce RTX 4080 16 GB. If anyone has such a graphic card or one similar in power, how quickly are photos generated for you, I'm interested in generating realistic photos of a basic size for a website, is it worth buying this card for work instead Colab+

**Comment by mashb1t** [comment]
V100 is similar if not a bit worse than a 3080 (running a 3080 with avg. 2,78it/s).
I also have access to a PC with a 4090 with avg. 5,41it/s.

From personal experience I'd suggest to go for the 4080 if you're planning to use it long-term and with privacy concerns as no data is transmitted to Google this way.

https://technical.city/de/video/Tesla-V100-PCIe-32-GB-vs-GeForce-RTX-4080
https://bizon-tech.com/gpu-benchmarks/NVIDIA-Tesla-V100-vs-NVIDIA-RTX-3080/535vs578

**Reply by quasar136a8** [reply]
What do you mean with "privacy concerns as no data is transmitted to Google this way."?
For the not advanced user, 4090 is much faster, but you recommend 4080, why?

**Reply by mashb1t** [reply]
@quasar136a8 i mean, you wrote that you're thinking about getting a 4080. Of course a 4090 is faster, but also 70% more expensive. Depending on your financial resources you can get the one or the other or try various cloud instances, e.g. eith an A100 or similar.

**Reply by mashb1t** [reply]
data privacy => Google knows what you've generated. If you are running a business you normally don't want some 3rd party to know what you're doing.

**Reply by quasar136a8** [reply]
You say the 4090 sending this date and 4080 no? 
How is this regulated on the graphic card?

**Reply by mashb1t** [reply]
bro, GPUs aren't sending any data to Google, but using Google Colab is obviously collecting data 😅

**Comment by quasar136a8** [comment]
If I go to 4080 or 4090 image generation works faster than colab?

To understand of process, I downloaded fooocus on pc, win 11, Rizen 9, RTX 1660super, 32GB ram, and a few minutes 4-5 when I started generating the images computer started to be very slow almost freezing, because fooocus downloading models, it download the same models each time or just once and after few generation of the images in the same syle its will be all much faster?
<img width="865" alt="Screenshot 2024-02-23 215407" src="https://github.com/lllyasviel/Fooocus/assets/111929719/205079b3-5f49-46d7-9142-9fb62bfa5f71">

**Reply by mashb1t** [reply, marked as answer]
Fooocus downloads the models once, and then reuses them so you don't have to download them again.

**Reply by quasar136a8** [reply]
If I go to 4080 image generation works faster than colab on T4?

**Reply by mashb1t** [reply]
yes, roughly 4x faster

**Reply by Amit30swgoh** [reply]
I have Colab+ even when mounting the drive with all models in it is kind of slow i must do something bad

**Reply by mashb1t** [reply]
@Amit30swgoh please check if https://github.com/lllyasviel/Fooocus/issues/2327#issuecomment-1963649657 helps in improving / better distributing the load

**Reply by Amit30swgoh** [reply]
Mash,
 perhaps you have a simple guide on how to mount the right way and run foocus on colab gradio using the Google drive? (Colab pro)..
It takes me around 5 min to get gradio share link
Thanks
